caluma/caluma_analytics/simple_table.py

```diff
@@ -86,9 +87,23 @@ def source_path(self) -> List[str]:
     def parse_value(self, value):
         """Parse value from DB into python value.
 
-        Only needs to be overloaded when required.
-        Used in GraphQL context only
+        Ensure date / time types are using the correct timezone,
+        and numeric types are int or float.
         """
+        if isinstance(value, Decimal):
+            # Postgres sometimes returns Decimal types that
+            # need to be converted for "proper" representation
+            # in the output
+            return float(value)  # pragma: no cover
```

Comment:
This seems like something that should be covered..

Reply:
Sadly, this depends on the postgres version - so this code path is not covered in all variants we test in CI at the moment

Reply:
Damn, makes sense. Maybe change the comment so this is clear